compiler/src/main_compiler.ml

```diff
@@ -171,25 +171,10 @@ let main () =
     visit_prog_after_pass ~debug:true Compiler.ParamsExpansion prog;
 
     if !ec_list <> [] || !ecfile <> "" then begin
-      let fmt, close =
-        if !ecfile = "" then Format.std_formatter, fun () -> ()
-        else
-          let out = open_out !ecfile in
-          let fmt = Format.formatter_of_out_channel out in
-          fmt, fun () -> close_out out in
-      let fnames =
-        match !ec_list with
-        | [] -> List.map (fun { f_name ; _ } -> f_name.fn_name) (snd prog)
-        | fnames -> fnames in
-      begin try
-        BatPervasives.finally
-          (fun () -> close ())
-          (fun () -> ToEC.extract Arch.reg_size Arch.asmOp fmt !model prog fnames)
-          ()
-      with e ->
-        BatPervasives.ignore_exceptions
-          (fun () -> if !ecfile <> "" then Unix.unlink !ecfile) ();
-        raise e end;
```

Comment:
Please leave this code here. The deprecation notice can go in the `CLI_errors` module.

It is wrong to use the `Unix` module in `ToEC`.

Reply:
done